### ISSUE TITLE
Citation request for "probabilities for each class" in the doc's description on cross-entropy

### ISSUE THREAD

**Issue by ZhenyueQin** [pytorch/pytorch#69363]
## 📚 Documentation

Hi PyTorch developers, 

Please first let me express my gratitude for PyTorch, a marvellous deep learning framework. 

I found that in the doc of PyTorch v 1.10.0, the probabilities for each class has been introduced in cross-entropy. The same form has been introduced in our 2019 paper: Rethinking Softmax with Cross-Entropy: Neural Network Classifier as Mutual Information Estimator (https://arxiv.org/abs/1911.10688). 

In the paper, we propose the probability-correct (PC) softmax. Its form is: 
$\sigma_p(n(\mathbf{x}))_y = \frac{ \exp( n(\mathbf{x})_y )}{\sum_{y'=1}^{M}P(y')\exp( n(\mathbf{x})_{y'})}$. 
It is Equation 10 of the paper. 

In the description of cross-entropy within PyTorch doc, the form is: 
$\ell(x, y) = L = \{l_1,\dots,l_N\}^\top, \quad l_n = - \sum_{c=1}^C w_c \log \frac{\exp(x_{n,c})}{\exp(\sum_{i=1}^C x_{n,i})} y_{n,c}$. 
The doc's link is: https://pytorch.org/docs/1.10.0/generated/torch.nn.CrossEntropyLoss.html?highlight=cross%20entropy#torch.nn.CrossEntropyLoss. 

We can see that the two forms are similar. We also describe the information-theoretic motivation behind this form. Hence, it may help people understand the rationale behind the form with probabilities. 

If possible, can you please consider citing our paper? The bibtex is as follows: 
@article{qin2019rethinking,
  title={Rethinking softmax with cross-entropy: Neural network classifier as mutual information estimator},
  author={Qin, Zhenyue and Kim, Dongwoo and Gedeon, Tom},
  journal={arXiv preprint arXiv:1911.10688},
  year={2019}
}

Thank you!

**Comment by albanD** [comment]
Note that the idea comes from the original proposal in https://github.com/pytorch/pytorch/issues/11959 not from this specific paper, so I am not sure we want to cite a paper for this.

**Comment by ZhenyueQin** [comment]
Thank you for the reply albanD. The paper interprets adding the probability from the information theory perspective. I believe it will help people understand the rationale from a new perspective. Can you please consider the citation again? Thank you very much.
